status/backend.json

```diff
@@ -24,6 +24,9 @@
       "enabled": true,
       "merchants_v2": false
     },
+    "fims": {
+      "enabled": true
```

Comment:
Is it ready?

Reply:
nope, it was only CP&Paste from another FF
```suggestion
      "enabled": false
```